### PR TITLE
change skip-layernorm to adapt a new method

### PR DESCRIPTION
**Pull request by ZhangHandi**
### PR types
New features

### PR changes
Others

### Describe
change try_skip_layernorm_fuse_pass.cc and skip_layernorm.cc to adapt a new compression method

### CODE REVIEW

File: paddle/fluid/inference/tensorrt/convert/skip_layernorm.cc [hunk 1]
```diff
@@ -133,6 +142,15 @@ class SkipLayerNormOpConverter : public OpConverter {
...
            static_cast<int32_t>(scale_weight.count)},
       };
+
+      if (use_smooth) {
+        type = static_cast<int32_t>(nvinfer1::DataType::kINT8);
+        fields.push_back({"smooth_scale",
+                          smooth_scale.data(),
+                          nvinfer1::PluginFieldType::kFLOAT32,
+                          static_cast<int32_t>(smooth_scale.size())});
+      }
```
Comment (Wangzheee) [inline paddle/fluid/inference/tensorrt/convert/skip_layernorm.cc:200]: if 后接 else

File: paddle/fluid/inference/tensorrt/convert/skip_layernorm.cc [hunk 2]
```diff
@@ -133,28 +142,61 @@ class SkipLayerNormOpConverter : public OpConverter {
...
+        nvinfer1::PluginFieldCollection* pluginPtr =
+            static_cast<nvinfer1::PluginFieldCollection*>(
+                malloc(sizeof(nvinfer1::PluginFieldCollection) +
+                       fields.size() *
+                           sizeof(nvinfer1::PluginField)));  // remember to free
+        pluginPtr->nbFields = static_cast<int32_t>(fields.size());
+        pluginPtr->fields = fields.data();
+
+        auto pluginObj = creator->createPlugin(
+            "CustomSkipLayerNormPluginDynamic", pluginPtr);
```
Comment (Wangzheee) [inline paddle/fluid/inference/tensorrt/convert/skip_layernorm.cc]: 这个名字和后面的区分一下？类似CustomSkipLayerNormPluginDynamicWithSmooth

File: paddle/fluid/inference/tensorrt/convert/skip_layernorm.cc [hunk 3]
```diff
@@ -133,28 +142,61 @@ class SkipLayerNormOpConverter : public OpConverter {
...
+        free(pluginPtr);
+
+        auto plugin_layer = engine_->network()->addPluginV2(
+            inputs.data(), inputs.size(), *pluginObj);
+
+        PADDLE_ENFORCE_NE(
+            plugin_layer,
+            nullptr,
+            platform::errors::InvalidArgument(
+                "fail to add CustomSkipLayerNormPluginDynamic layer"));
```
Comment (Wangzheee) [inline paddle/fluid/inference/tensorrt/convert/skip_layernorm.cc]: 同上 CustomSkipLayerNormPluginDynamicWithSmooth

**Review by ceci3 — APPROVED** [review]
LGTM

**Pull request merged**